docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -48,3 +48,58 @@ If your issue has not been resolved by this procedure, perform the following ste
 . link:https://github.com/code-ready/crc/issues[Search open issues] for the issue that you are encountering.
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
+
+= Differences with a production OpenShift install
+
+{rh-prod} is a regular OpenShift install tweaked to be fit in a single VM on a laptop.
```

Comment:
Rather than "tweaked", could we say "configured"? Also, do we have to specify "laptop"?

```adoc
{rh-prod} is a regular OpenShift installation configured to fit in a single virtual machine.
```

Reply:
I think what I was trying to underline is that it's not a big VM, and that it's meant for development purposes. Maybe
`{rh-prod} is a regular OpenShift install configured to fit in a single virtual machine meant to be run on a developer's machine.`?